environment/src/main/java/com/sequenceiq/environment/network/service/NetworkCreationRequestFactory.java

```diff
@@ -70,4 +77,13 @@ private boolean getPrivateSubnetEnabled(EnvironmentDto environmentDto) {
     private String getUserFromCrn(String crn) {
         return Optional.ofNullable(Crn.fromString(crn)).map(Crn::getUserId).orElse(null);
     }
+
+    private CDPTagMergeRequest cdpTagMergeRequest(EnvironmentDto environmentDto) {
```

Comment:
I sow a similar method, maybe we should create a dedicated factory or constructor for creating this object from EnvironmentDto.